grafonnet/bar_gauge_panel.libsonnet

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
It is not formatted well but I did not find a way how to install the formatter.

Reply:
Does this help?

```
go get github.com/google/go-jsonnet/cmd/jsonnetfmt
```